src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -523,7 +523,7 @@ private String resolveContent(String result, Set<String> usedIds) {
      */
     public static Optional<String> getResolvedField(String field, BibEntry entry, BibDatabase database) {
         Objects.requireNonNull(entry, "entry cannot be null");
-        if ("bibtextype".equals(field)) {
+        if (BibEntry.TYPE_HEADER.equals(field) || "bibtextype".equals(field)) {
```

Comment:
Why do we need two different strings for that? (`entryType` and `bibtextype`)?

When searching the code, only `BibEntry.TYPE_HEADER` should be sufficient. However, it might be the case that my search was too straight-forward.

![grabbed_20160815-081924](https://cloud.githubusercontent.com/assets/1366654/17657095/feeb515e-62c0-11e6-8c85-b616a798a495.png)